autopush/web/webpush.py

```diff
@@ -298,13 +298,35 @@ def token_prep(self, d):
         )
         return d
 
+    def vapid_draft_01(self, token, public_key):
+        try:
+            jwt = extract_jwt(token, public_key)
+        except (AssertionError, ValueError, JOSEError):
+            raise InvalidRequest("Invalid Authorization Header",
+                                 status_code=401, errno=109,
+                                 headers={"www-authenticate": PREF_SCHEME})
+        return jwt
+
+    def vapid_draft_02(self, rawtoken):
+        try:
+            tokens = rawtoken.split(',')
+            elements = dict()
+            for token in tokens:
+                kv = token.replace(' ', '').split('=')
```

Comment:
probably want maxsplit of 1 to be safe, like parse_endpoint does. kv[1] can potentially raise an IndexError too (even with a maxsplit)